### ISSUE TITLE
Can not receive file(image or other format)

### ISSUE THREAD

**Issue by ChenMoGe2** [tigase/stork#36]
**Describe the bug**
Can not receive file sended by others

**To Reproduce**
1. Go to 'one of chat item'
2. Click on 'Send file'
3. Choose a photo
4. I can just see the photo on my phone,but the account who I'm chatting with can not receive the file I sended.

**Expected behavior**
Can receive file sended by others

**Smartphone (please complete the following information):**
 - Device: Oneplus 7 Pro
 - OS: Android 9,10

**Comment by licaon-kter** [comment]
Random question, why use Stork? Yes it works but it's lacking so much compared to Conversations, imho :)

**Comment by ChenMoGe2** [comment]
@licaon-kter Thank you
What else is there besides the stork can support xmpp at Android system

**Comment by Yoyosif** [comment]
Hello [tigase/stork]
I need Auto response for the incoming messages in main chatroom 😭❤️
I want to make bot in my Own chatroom
Please help me 🙏😘🌹

في الأربعاء، ٢١ أبريل ٢٠٢١ ١٢:٢٧ م ChenMoGe2 ***@***.***> كتب:

**Comment by licaon-kter** [comment]
@ChenMoGe2 like I said: Conversations. You find it lacking?

**Comment by woj-tek** [comment]
@ChenMoGe2 where the file correctly upload to the server (I assume http-upload was used)?

@Yoyosif 
Please submit dedicated issue. Though, depending on the use case, it could be handled better by the server-side component.

@licaon-kter personal preference? :D let's keep it on-topic? :-)

**Comment by ChenMoGe2** [comment]
@woj-tek Thanks
I will try again

**Comment by ChenMoGe2** [comment]
@licaon-kter 
.......

**Comment by ChenMoGe2** [comment]
@licaon-kter Thanks
I understand your answer.Conversations is an application.
Does it support multi-user video or audio chat?

**Comment by licaon-kter** [comment]
@ChenMoGe2 no, only 1:1, compatible with Movim, Dino and Siskin, but not sure if with Stork too. There's no definitive standard for group a/v yet.

Better install a Jitsi Meet instance alongside, it's still XMPP.

**Comment by ChenMoGe2** [comment]
@licaon-kter 
OK,Thanks.I will try the Jitsi Meet.I just want to integrate them on one application
